test/common/common/utility_speed_test.cc

```diff
@@ -246,8 +246,8 @@ static void BM_IntervalSetInsert17(benchmark::State& state) {
     interval_set.insert(3, 6);
     interval_set.insert(3, 20);
     interval_set.insert(3, 22);
-    interval_set.insert(-2, 23);
-    interval_set.insert(-3, 24);
+    interval_set.insert(23, 9223372036854775806UL);
+    interval_set.insert(24, 9223372036854775805UL);
```

Comment:
Why is this being changed?

Reply:
size_t is an unsigned type.  -2 ends up being interpreted as 9223372036854775806UL

interval set requires insert arguments to be sorted in value, so interval_set.insert(-2, 23), also known as interval_set.insert(9223372036854775806UL, 23) triggers an ASSERT failure in debug modes.